Passwordstate dynamic folder

dnewsholme opened the pull request:
Adds a dynamic folder for PasswordState bringing in passwords only.

StefanKoell commented:
first of all, thx for your contribution! We very much appreciate it!

Before we go ahead and merge your PR, we'd like to clarify some things and would like to request some improvements.

1. Is this a Windows-only script or does it work with PowerShell Core across platforms? If it's Windows-only or you have only tested it on Windows, please mention that fact in the folder name and/or the dynamic folder export file's name. Something like "Dynamic Folder Name (Windows only) should work. 

2. Your export file doesn't contain any setup or configuration instruction for the user. Please see how we did this in other samples. You can use the "Notes" field to provide setup/config instructions and the notes section will automatically be opened once a user imports your dynamic folder definition.

Thx!

dnewsholme commented:
It's Powershell core compatible and runs cross OS. I've updated the Description to show this and added a notes section.

Let me know if you require any other changes.

Thanks

StefanKoell commented:
Fantastic! Thanks for the changes!
